model/Software/Properties/gitoid.md

```diff
@@ -0,0 +1,17 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# gitoid
+
+## Summary
+
+Git Object ID following the IANA gitoid URI scheme.
+
+## Description
+
+https://www.iana.org/assignments/uri-schemes/prov/gitoid Gitoid stands for [Git Object ID](https://git-scm.com/book/en/v2/Git-Internals-Git-Objects) and a gitoid of type blob is a unique hash of a binary artifact. 
```

Comment:
> ... of type blob is a unique hash of a binary artifact.

we also need to accept `tree` here, right?

Reply:
For the `tree` variant, we might need to define a way, which files were included and relative to which root is was computed. Otherwise it is hard to reproduce. Especially if the SBOM is committed to the source tree.

Reply:
> For the tree variant, we might need to define a way, which files were included and relative to which root is was computed. Otherwise it is hard to reproduce. Especially if the SBOM is committed to the source tree.

Sounds like we're re-inventing the `PackageVerificationCode`.  Perhaps if there are excluded files, we recommend the `PackageVerificationCode`

Reply:
> we also need to accept tree here, right?

Yes - @maxhbr  can you propose some specific language?

Reply:
```suggestion
https://www.iana.org/assignments/uri-schemes/prov/gitoid Gitoid stands for [Git Object ID](https://git-scm.com/book/en/v2/Git-Internals-Git-Objects) and a gitoid is a unique hash of a binary artifact. 
```